pkg/api/v1beta1/types.go

```diff
@@ -1569,6 +1569,8 @@ const (
 	// DNSDefault indicates that the pod should use the default (as
 	// determined by kubelet) DNS settings.
 	DNSDefault DNSPolicy = "Default"
+
+	DefaultTerminationGracePeriodSeconds = 30
```

Comment:
Please document the default value in the description tag of TerminationGracePeriodSeconds in all API versions.